### PR TITLE
Configurable stack analysis manifest

### PR DESCRIPTION
**Pull request by tisnik**
Support for the following call:

```
./runtest.sh -S --manifest=vertx_3_4_2.xml
```

ie. it is possible to specify manifest file that will be sent for the stack analysis.

### CODE REVIEW

**Comment by tisnik** [comment]
[test this]

File: perf-tests/src/coreapi.py [hunk 1]
```diff
@@ -31,13 +41,39 @@ def contains_alternate_node(json_resp):
...
+    def prepare_manifest_files(filename):
+        """Send the selected manifest file to stack analysis."""
+        # default variable substitutions
+        filename = filename or 'requirements_click_6_star.txt'
+        manifest_name = CoreApi.get_manifest_name(filename)
+
+        print("filename with manifest data: {m}".format(m=manifest_name))
+        print("standard manifest name: {n}".format(n=manifest_name))
+
+        filename = 'data/{filename}'.format(filename=filename)
```
Comment (pkajaba) [inline perf-tests/src/coreapi.py]: this means that manifest files have to be stored always inside `data` folder. Is it your intention?

File: perf-tests/src/coreapi.py [hunk 2]
```diff
@@ -31,13 +41,39 @@ def contains_alternate_node(json_resp):
...
+        raise "Unknown extension in filename: {f}".format(f=filename)
+
+    @staticmethod
+    def prepare_manifest_files(filename):
+        """Send the selected manifest file to stack analysis."""
+        # default variable substitutions
+        filename = filename or 'requirements_click_6_star.txt'
+        manifest_name = CoreApi.get_manifest_name(filename)
+
+        print("filename with manifest data: {m}".format(m=manifest_name))
```
[the remark this answers is not in the document]
Comment (tisnik) [reply perf-tests/src/coreapi.py]: oh, good catch, TY

File: perf-tests/src/coreapi.py (hunk 1 above)
Comment (tisnik) [reply perf-tests/src/coreapi.py]: yes, to make perf.tests easier to run

**Pull request merged**